# Análisis de la subocupación en Argentina 1er trimestre 2021
**Por Violeta Carrera Pereyra** 


## Objetivo
Ponerse tres preguntas
- ZONAS GEOGRAFICAS
- SEXO/NIVEL EDUCATIVO
- TEMPORAL? (´COMO CAMBIO CON PANDEMIA) -- NEC JUNTAR DATASETS PRE PANDEMIA (2017-2021) son 15 datasets en total DUDAS 

## Tabla de contenidos
1. Carga de datos
2. Exploración de datos
3. Limpieza de datos
4. Presentación de datos
5. Conclusiones
6. Referencias





# 1. Carga de datos
Carga de la base (Encuesta Permanente de Hogares del INDEC primer trimestre del 2021) y recorte de las columnas con las que se trabajará.



In [1]:
#Importar librerias
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt 
%matplotlib inline


In [2]:
#Importar datos
import os 
print(os.getcwd())

C:\Users\User\Documents\GitHub\EPH-subocupaci-n


In [3]:
eph_completa = pd.read_excel("data/usu_individual_T121.xlsx")
eph_completa.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 46693 entries, 0 to 46692
Columns: 177 entries, CODUSU to PONDIH
dtypes: float64(115), int64(59), object(3)
memory usage: 63.1+ MB


In [4]:
print(list(eph_completa.columns))


['CODUSU', 'ANO4', 'TRIMESTRE', 'NRO_HOGAR', 'COMPONENTE', 'H15', 'REGION', 'MAS_500', 'AGLOMERADO', 'PONDERA', 'CH03', 'CH04', 'CH05', 'CH06', 'CH07', 'CH08', 'CH09', 'CH10', 'CH11', 'CH12', 'CH13', 'CH14', 'CH15', 'CH15_COD', 'CH16', 'CH16_COD', 'NIVEL_ED', 'ESTADO', 'CAT_OCUP', 'CAT_INAC', 'IMPUTA', 'PP02C1', 'PP02C2', 'PP02C3', 'PP02C4', 'PP02C5', 'PP02C6', 'PP02C7', 'PP02C8', 'PP02E', 'PP02H', 'PP02I', 'PP03C', 'PP03D', 'PP3E_TOT', 'PP3F_TOT', 'PP03G', 'PP03H', 'PP03I', 'PP03J', 'INTENSI', 'PP04A', 'PP04B_COD', 'PP04B1', 'PP04B2', 'PP04B3_MES', 'PP04B3_ANO', 'PP04B3_DIA', 'PP04C', 'PP04C99', 'PP04D_COD', 'PP04G', 'PP05B2_MES', 'PP05B2_ANO', 'PP05B2_DIA', 'PP05C_1', 'PP05C_2', 'PP05C_3', 'PP05E', 'PP05F', 'PP05H', 'PP06A', 'PP06C', 'PP06D', 'PP06E', 'PP06H', 'PP07A', 'PP07C', 'PP07D', 'PP07E', 'PP07F1', 'PP07F2', 'PP07F3', 'PP07F4', 'PP07F5', 'PP07G1', 'PP07G2', 'PP07G3', 'PP07G4', 'PP07G_59', 'PP07H', 'PP07I', 'PP07J', 'PP07K', 'PP08D1', 'PP08D4', 'PP08F1', 'PP08F2', 'PP08J1', 'PP

In [4]:
#Recorte de la base
eph = eph_completa[["CODUSU", "REGION", "AGLOMERADO", "CH04", "CH06", "NIVEL_ED", "CAT_OCUP", "PP03C", "INTENSI", "PP07A", "P47T", "DECINDR", "ITF", "DECIFR", "PP07H", "ESTADO"]]

In [4]:
#Transformación del INDICE por variable CODUSU (código de identificación de los entrevistados que permite, eventualmente, vincular la base individual con la de hogares)
#eph.set_index("CODUSU",inplace = True)

# 2. Exploración de datos


In [6]:
eph.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 46693 entries, 0 to 46692
Data columns (total 17 columns):
 #   Column      Non-Null Count  Dtype  
---  ------      --------------  -----  
 0   CODUSU      46693 non-null  object 
 1   REGION      46693 non-null  int64  
 2   AGLOMERADO  46693 non-null  int64  
 3   CH03        46693 non-null  int64  
 4   CH04        46693 non-null  int64  
 5   CH06        46693 non-null  int64  
 6   NIVEL_ED    46693 non-null  int64  
 7   CAT_OCUP    46693 non-null  int64  
 8   PP03C       19014 non-null  float64
 9   INTENSI     19014 non-null  float64
 10  PP07A       19014 non-null  float64
 11  P47T        46619 non-null  float64
 12  DECINDR     46693 non-null  int64  
 13  ITF         46691 non-null  float64
 14  DECIFR      46691 non-null  float64
 15  PP07H       19014 non-null  float64
 16  ESTADO      46693 non-null  int64  
dtypes: float64(7), int64(9), object(1)
memory usage: 6.1+ MB


In [7]:
eph.sample(5)

CODUSU  REGION  AGLOMERADO  CH03  CH04  CH06  \
29298  TQRMNORQUHMMKNCDEHLEH00699156      40          25     5     1    23   
22223  TQRMNOQQTHLOLPCDEGPDJ00648969      40          19     9     2    40   
10195  TQRMNOSQRHLOKUCDEFPCH00647755      44           9     5     1    -1   
10342  TQRMNOTQQHKOLQCDEFPCH00653181      44           9     1     1    60   
9644   TQRMNOQRQHLNKTCDEFPCH00647845      44           9     2     2    58   

       NIVEL_ED  CAT_OCUP  PP03C  INTENSI  PP07A     P47T  DECINDR       ITF  \
29298         5         0    NaN      NaN    NaN      0.0        0  110000.0   
22223         4         2    1.0      1.0    0.0  17000.0        3   37000.0   
10195         7         0    NaN      NaN    NaN      0.0        0   82000.0   
10342         4         3    1.0      3.0    6.0  66000.0        9  255500.0   
9644          4         0    NaN      NaN    NaN  30000.0        6  127000.0   

       DECIFR  PP07H  ESTADO  
29298     9.0    NaN       3  
22223     3.0    0.0       1  
10195     7.0    NaN       4  
10342    10.0    1.0       1  
9644      9.0    NaN       3

In [8]:
eph.shape

(46693, 17)

In [9]:
#Valores únicos y frecuencia relativa de cada columna del dataset (aplicado solo a variables categóricas y edad)
#Región geográfica
print(eph.REGION.unique())
eph.REGION.value_counts(dropna=False, normalize=True)

[43 41 44 42 40  1]


43    0.307412
40    0.240914
44    0.139892
42    0.117084
41    0.107275
1     0.087422
Name: REGION, dtype: float64

In [11]:
#Aglomerado urbano
print(eph.AGLOMERADO.unique())
eph.AGLOMERADO.value_counts(dropna=False, normalize=True)

[ 2  3  4  5  6  7  8  9 10 12 13 14 15 17 18 19 20 22 23 25 26 27 29 30
 31 32 33 34 36 38 91 93]


33    0.066070
23    0.054890
10    0.050179
29    0.043583
4     0.039321
19    0.038807
27    0.038357
18    0.036858
14    0.035980
13    0.035701
25    0.034288
5     0.032553
22    0.032489
38    0.032146
15    0.031739
6     0.029276
26    0.028548
7     0.028141
12    0.026813
91    0.026685
2     0.026385
17    0.023986
9     0.023580
93    0.023451
3     0.022916
36    0.021952
32    0.021352
20    0.021245
31    0.020945
8     0.020581
34    0.020239
30    0.010944
Name: AGLOMERADO, dtype: float64

In [12]:
#Sexo
print(eph.CH04.unique())
eph.CH04.value_counts(dropna=False, normalize=True)

[2 1]


2    0.522005
1    0.477995
Name: CH04, dtype: float64

In [96]:
#Edad
print(eph.CH06.unique())
eph.CH06.value_counts(dropna=False, normalize=True)

[ 27  26  38  33  15  17  10  25  21  49  37  62  34  28  60  69  70  74
  44  54  50  19  68  67  58  61  75  56  59  42  40  16  14  64  30  32
  55  24  22  18  11   7  43  47   9  35   6  63  45  57  46   4  53  20
  36  41  51  52  91  72   8  29  31  77  66  23  80  81  48   5  12   3
   2  13  84  82  71  65  76  73  78  79  -1   1  85  39  89  90  83  97
  92  87  88  86  98  93  94  99  96  95 101]


20     0.017454
16     0.016705
21     0.016683
17     0.016448
23     0.016448
         ...   
95     0.000150
97     0.000128
98     0.000107
99     0.000107
101    0.000021
Name: CH06, Length: 101, dtype: float64

In [13]:
#Nivel educativo
print(eph.NIVEL_ED.unique())
eph.NIVEL_ED.value_counts(dropna=False, normalize=True)

[6 5 3 1 2 4 7]


3    0.209582
4    0.198252
1    0.142291
6    0.129848
2    0.123487
5    0.120703
7    0.075836
Name: NIVEL_ED, dtype: float64

In [14]:
#Categoría ocupacional
print(eph.CAT_OCUP.unique())
eph.CAT_OCUP.value_counts(dropna=False, normalize=True)

[3 0 2 1 4 9]


0    0.563703
3    0.314287
2    0.103806
1    0.015141
4    0.003020
9    0.000043
Name: CAT_OCUP, dtype: float64

In [99]:
#Cantidad de empleos
print(eph.PP03C.unique())
eph.PP03C.value_counts(dropna=False, normalize=True)

[ 1. nan  2.  0.]


NaN    0.592787
1.0    0.338680
0.0    0.039985
2.0    0.028548
Name: PP03C, dtype: float64

In [123]:
#Estado de ocupación
print(eph.ESTADO.unique())
eph.ESTADO.value_counts(dropna=False)

[1 3 2 4 0]


3    19684
1    19014
4     6155
2     1766
0       74
Name: ESTADO, dtype: int64

In [125]:
#Itensidad de la ocupación
print(eph.INTENSI.unique())
eph.INTENSI.value_counts(dropna=False, normalize=True)

[ 3. nan  1.  4.  2.]


NaN    0.592787
2.0    0.208725
3.0    0.109374
1.0    0.049108
4.0    0.040006
Name: INTENSI, dtype: float64

In [104]:
#Hace cuánto trabajan en la ocupación principal
print(eph.PP07A.unique())
eph.PP07A.value_counts(dropna=False, normalize=True)

[ 5. nan  2.  0.  6.  4.  3.  9.  1.]


NaN    0.592787
6.0    0.150365
0.0    0.141563
5.0    0.073930
2.0    0.012529
4.0    0.011522
3.0    0.009637
1.0    0.007003
9.0    0.000664
Name: PP07A, dtype: float64

In [106]:
#Deciles de ingreso total
print(eph.DECINDR.unique())
eph.DECINDR.value_counts(dropna=False, normalize=True)

[ 7  4 12  2  0  3  1  5  6  9 10  8 13]


0     0.388902
12    0.081554
3     0.065277
2     0.063414
5     0.056839
6     0.056582
4     0.056197
7     0.051121
1     0.050264
8     0.045510
9     0.044482
10    0.038314
13    0.001542
Name: DECINDR, dtype: float64

In [108]:
#Deciles de ingreso familiar
print(eph.DECIFR.unique())
eph.DECIFR.value_counts(dropna=False, normalize=True)

[ 4.  2. 12.  1.  9.  8.  5. 10.  6.  7.  3.  0. nan]


12.0    0.180734
7.0     0.099308
4.0     0.092326
3.0     0.085944
5.0     0.085216
6.0     0.084167
8.0     0.083524
9.0     0.082346
2.0     0.076521
10.0    0.066670
1.0     0.059923
0.0     0.003277
NaN     0.000043
Name: DECIFR, dtype: float64

In [109]:
#Descuento por jubilación
print(eph.PP07H.unique())
eph.PP07H.value_counts(dropna=False, normalize=True)

[ 1. nan  2.  0.]


NaN    0.592787
1.0    0.192727
0.0    0.114021
2.0    0.100465
Name: PP07H, dtype: float64

In [114]:
#Resúmenes estadísticos de variables de razón 
#CH06 = AÑOS
eph.CH06.describe()

count    46693.000000
mean        35.819952
std         22.439663
min         -1.000000
25%         17.000000
50%         33.000000
75%         53.000000
max        101.000000
Name: CH06, dtype: float64

In [117]:
#P47T = monto del ingreso total
eph.P47T.describe()

count     46619.000000
mean      19814.440786
std       31705.829005
min          -9.000000
25%           0.000000
50%        6000.000000
75%       30000.000000
max      819000.000000
Name: P47T, dtype: float64

In [119]:
#ITF= monto del ingreso familiar
eph.ITF.describe()

count     46691.000000
mean      61272.261742
std       59675.746294
min           0.000000
25%       20800.000000
50%       50000.000000
75%       84000.000000
max      819000.000000
Name: ITF, dtype: float64

De la lectura de los outputs correspondientes a esta sección se desprende una constatación central: la presencia de la misma cantidad de datos nulos (27679) en 5 columnas (PP03C, INTENSI, PP04D, PP07A y PP07H). Luego de realizar el value_counts de la columna del dataset ESTADO esta particularidad de la base cobra sentido: los datos no nulos de las 5 columnas corresponden a los ocupados (o valor 1 de la columna ESTADO). Es decir, son todas preguntas que solamente se le hicieron a las personas ocupadas. Como en la proxima sección se filtraran solamente los casos de los ocupados estos valores nulos no resultan un problema.
Sí parece haber habido un error en la carga de datos en la variable edad. Al ver el mínimo de la columna y sus valores únicos aparece el -1 como un valor dispuesto. Habrá que solucionar este problema en la limpieza de datos. 
Cabe mencionar, también, el valor mínimo de P47T (monto total de ingresos indivudales). El -9 y el 9, 99 y 999 en el resto de las variables se corresponde con el "No sabe/no contesta". Algo similar sucede con el 12 y 13 que aparecen como valores únicos en DECIFR y DECINDR (variables que versan sobre los deciles de ingresos), implican que no se respondio a la pregunta o que no se hizo la entrevista individual, respectivamente.
Después no encuentro anomalías: todos los valores que adquieren las variables estan contemplados en el manual de la EPH y en la próxima sección  se procederá a etiquetarlos para que los datos puedan cobrar cierto sentido. 

# 3. Limpieza de datos 
Se realizará primero un recorte de la base y se etiquetaran los valores y las variables. Una vez recortada no deberían quedar valores nulos (por la correspondencia de estos con las personas no ocupadas). A su vez, se procederá a eliminar el caso con edad = -1 y a guardar la base limpia.


In [63]:
#Renombre de columnas
renombrada = eph.rename(columns = {'CH04': 'SEXO', 'CH06': 'AÑOS', "NIVEL_ED": "NIVEL_EDUCATIVO", "CAT_OCUP": "CATEGORÍA_OCUPACIONAL", "PP03C": "CANTIDAD_DE_TRABAJOS", "PP07A": "ANTIGUEDAD_EN_OCUPACIÓN", "P47T": "MONTO_DE_INGRESO_TOTAL", "DECINDR": "DECIL_DE_INGRESOS", "ITF": "INGRESO_TOTAL_FAMILIAR", "DECIFR": "DECILES_SEGÚN_INGRESO_FAMILIAR", "PP07H": "DESCUENTO_POR_JUBILACIÓN", "INTENSI": "INTENSIDAD_DE_LA_OCUPACIÓN", "ESTADO": "ESTADO_DE_OCUPACIÓN"}, inplace = False)


In [37]:
renombrada.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 46693 entries, 0 to 46692
Data columns (total 16 columns):
 #   Column                          Non-Null Count  Dtype 
---  ------                          --------------  ----- 
 0   CODUSU                          46693 non-null  object
 1   REGION                          46693 non-null  object
 2   AGLOMERADO                      46693 non-null  object
 3   SEXO                            46693 non-null  object
 4   AÑOS                            46693 non-null  object
 5   NIVEL_EDUCATIVO                 46693 non-null  object
 6   CATEGORÍA_OCUPACIONAL           46693 non-null  object
 7   CANTIDAD_DE_TRABAJOS            46693 non-null  object
 8   INTENSIDAD_DE_LA_OCUPACIÓN      46693 non-null  object
 9   ANTIGUEDAD_EN_OCUPACIÓN         46693 non-null  object
 10  MONTO_DE_INGRESO_TOTAL          46693 non-null  object
 11  DECIL_DE_INGRESOS               46693 non-null  object
 12  INGRESO_TOTAL_FAMILIAR          46693 non-null

In [85]:
#Renombre de valores 
renombrada.loc[renombrada.REGION == 1, "REGION"] = "Gran Buenos Aires"
renombrada.loc[renombrada.REGION == 40, "REGION"] = "NOA"
renombrada.loc[renombrada.REGION == 41, "REGION"] = "NEA"
renombrada.loc[renombrada.REGION == 42, "REGION"] = "Cuyo"
renombrada.loc[renombrada.REGION == 43, "REGION"] = "Pampeana"
renombrada.loc[renombrada.REGION == 44, "REGION"] = "Patagonia"
renombrada.loc[renombrada.SEXO == 1, "SEXO"] = "Varón"
renombrada.loc[renombrada.SEXO == 2, "SEXO"] = "Mujer"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 1, "NIVEL_EDUCATIVO"] = "Primario incompleto"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 2, "NIVEL_EDUCATIVO"] = "Primario completo"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 3, "NIVEL_EDUCATIVO"] = "Secundario incompleto"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 4, "NIVEL_EDUCATIVO"] = "Secundario completo"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 5, "NIVEL_EDUCATIVO"] = "Superior universitario incompleto"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 6, "NIVEL_EDUCATIVO"] = "Superior universitario completo"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 7, "NIVEL_EDUCATIVO"] = "Sin instrucción"
renombrada.loc[renombrada.NIVEL_EDUCATIVO == 9, "NIVEL_EDUCATIVO"] = "Ns/Nc"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 0, "CATEGORÍA_OCUPACIONAL"] = "No le corresponde la secuencia analizada"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 1, "CATEGORÍA_OCUPACIONAL"] = "Patrón"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 2, "CATEGORÍA_OCUPACIONAL"] = "Cuenta propia"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 3, "CATEGORÍA_OCUPACIONAL"] = "Obrero o empleado"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 4, "CATEGORÍA_OCUPACIONAL"] = "Trabajador familiar sin remuneración"
renombrada.loc[renombrada.CATEGORÍA_OCUPACIONAL == 9, "CATEGORÍA_OCUPACIONAL"] = "Ns/Nc"
renombrada.loc[renombrada.CANTIDAD_DE_TRABAJOS == 1, "CANTIDAD_DE_TRABAJOS"] = "Un solo empleo o actividad"
renombrada.loc[renombrada.CANTIDAD_DE_TRABAJOS == 2, "CANTIDAD_DE_TRABAJOS"] = "Más de un empleo o actividad"
renombrada.loc[renombrada.CANTIDAD_DE_TRABAJOS == 0, "CANTIDAD_DE_TRABAJOS"] = "No le corresponde la secuencia analizada"
renombrada.loc[renombrada.INTENSIDAD_DE_LA_OCUPACIÓN == 1, "INTENSIDAD_DE_LA_OCUPACIÓN"] = "Subocupación por insuficiencia horaria"
renombrada.loc[renombrada.INTENSIDAD_DE_LA_OCUPACIÓN == 2, "INTENSIDAD_DE_LA_OCUPACIÓN"] = "Ocupado pleno"
renombrada.loc[renombrada.INTENSIDAD_DE_LA_OCUPACIÓN == 3, "INTENSIDAD_DE_LA_OCUPACIÓN"] = "Sobreocupado"
renombrada.loc[renombrada.INTENSIDAD_DE_LA_OCUPACIÓN == 4, "INTENSIDAD_DE_LA_OCUPACIÓN"] = "Ocupado que no trabajó en la semana"
renombrada.loc[renombrada.INTENSIDAD_DE_LA_OCUPACIÓN == 9, "INTENSIDAD_DE_LA_OCUPACIÓN"] = "Ns/Nc"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 1, "ANTIGUEDAD_EN_OCUPACIÓN"] = "No le corresponde la secuencia analizada"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 1, "ANTIGUEDAD_EN_OCUPACIÓN"] = "Menos de un mes"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 2, "ANTIGUEDAD_EN_OCUPACIÓN"] = "1 a 3 meses"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 3, "ANTIGUEDAD_EN_OCUPACIÓN"] = "más de 3 meses a 6 meses"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 4, "ANTIGUEDAD_EN_OCUPACIÓN"] = "más de 6 a 12 meses"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 5, "ANTIGUEDAD_EN_OCUPACIÓN"] = "más de 1 a 5 años"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 6, "ANTIGUEDAD_EN_OCUPACIÓN"] = "más de 5 años"
renombrada.loc[renombrada.ANTIGUEDAD_EN_OCUPACIÓN == 9, "ANTIGUEDAD_EN_OCUPACIÓN"] = "Ns/Nc"
renombrada.loc[renombrada.DECIL_DE_INGRESOS == 12, "DECIL_DE_INGRESOS"] = "No hay respuesta de ingresos"
renombrada.loc[renombrada.DECIL_DE_INGRESOS == 13, "DECIL_DE_INGRESOS"] = "Entrevista individual no analizada"
renombrada.loc[renombrada.DECILES_SEGÚN_INGRESO_FAMILIAR == 12, "DECILES_SEGÚN_INGRESO_FAMILIAR"] = "No hay respuesta de ingresos"
renombrada.loc[renombrada.DECILES_SEGÚN_INGRESO_FAMILIAR == 13, "DECILES_SEGÚN_INGRESO_FAMILIAR"] = "Entrevista individual no realizada"
renombrada.loc[renombrada.DESCUENTO_POR_JUBILACIÓN == 1,"DESCUENTO_POR_JUBILACIÓN"] = "Sí"
renombrada.loc[renombrada.DESCUENTO_POR_JUBILACIÓN == 1,"DESCUENTO_POR_JUBILACIÓN"] = "No"
renombrada.loc[renombrada.DESCUENTO_POR_JUBILACIÓN == 1,"DESCUENTO_POR_JUBILACIÓN"] = "No le corresponde la secuencia analizada"


In [86]:
renombrada.sample(5)

CODUSU             REGION  AGLOMERADO   SEXO  \
34739  TQRMNORYVHJOLPCDEHPJB00687675                NOA          29  Varón   
4714   TQRMNOQURHJMKMCDEFLID00668265           Pampeana           5  Mujer   
6051   TQRMNOPYSHKMLMCDEFMDB00652841           Pampeana           6  Mujer   
37134  TQRMNOQSPHJMNMCDEIIAD00690117  Gran Buenos Aires          32  Mujer   
11015  TQRMNOPVWHKLKUCDEGGFB00653419               Cuyo          10  Mujer   

       AÑOS                    NIVEL_EDUCATIVO  \
34739    13              Secundario incompleto   
4714     50  Superior universitario incompleto   
6051     35  Superior universitario incompleto   
37134    61    Superior universitario completo   
11015    62    Superior universitario completo   

                          CATEGORÍA_OCUPACIONAL        CANTIDAD_DE_TRABAJOS  \
34739  No le corresponde la secuencia analizada                         NaN   
4714                          Obrero o empleado  Un solo empleo o actividad   
6051                          Obrero o empleado  Un solo empleo o actividad   
37134  No le corresponde la secuencia analizada                         NaN   
11015  No le corresponde la secuencia analizada                         NaN   

      INTENSIDAD_DE_LA_OCUPACIÓN ANTIGUEDAD_EN_OCUPACIÓN  \
34739                        NaN                     NaN   
4714               Ocupado pleno           más de 5 años   
6051               Ocupado pleno           más de 5 años   
37134                        NaN                     NaN   
11015                        NaN                     NaN   

       MONTO_DE_INGRESO_TOTAL             DECIL_DE_INGRESOS  \
34739                     0.0                             0   
4714                     -9.0  No hay respuesta de ingresos   
6051                  36000.0                             6   
37134                    -9.0  No hay respuesta de ingresos   
11015                 27000.0                             5   

       INGRESO_TOTAL_FAMILIAR DECILES_SEGÚN_INGRESO_FAMILIAR  \
34739                 17400.0                            1.0   
4714                      0.0   No hay respuesta de ingresos   
6051                  75000.0                            7.0   
37134                     0.0   No hay respuesta de ingresos   
11015                 54000.0                            5.0   

      DESCUENTO_POR_JUBILACIÓN  ESTADO_DE_OCUPACIÓN  
34739                      NaN                    3  
4714                        Sí                    1  
6051                        Sí                    1  
37134                      NaN                    3  
11015                      NaN                    3

In [87]:
#Recorte de la base (solamente ocupados)
estado = renombrada[renombrada.ESTADO_DE_OCUPACIÓN == 1]
estado.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 19014 entries, 0 to 46687
Data columns (total 16 columns):
 #   Column                          Non-Null Count  Dtype  
---  ------                          --------------  -----  
 0   CODUSU                          19014 non-null  object 
 1   REGION                          19014 non-null  object 
 2   AGLOMERADO                      19014 non-null  int64  
 3   SEXO                            19014 non-null  object 
 4   AÑOS                            19014 non-null  int64  
 5   NIVEL_EDUCATIVO                 19014 non-null  object 
 6   CATEGORÍA_OCUPACIONAL           19014 non-null  object 
 7   CANTIDAD_DE_TRABAJOS            19014 non-null  object 
 8   INTENSIDAD_DE_LA_OCUPACIÓN      19014 non-null  object 
 9   ANTIGUEDAD_EN_OCUPACIÓN         19014 non-null  object 
 10  MONTO_DE_INGRESO_TOTAL          19014 non-null  float64
 11  DECIL_DE_INGRESOS               19014 non-null  object 
 12  INGRESO_TOTAL_FAMILIAR          

In [88]:
print(estado.isnull().sum())

CODUSU                            0
REGION                            0
AGLOMERADO                        0
SEXO                              0
AÑOS                              0
NIVEL_EDUCATIVO                   0
CATEGORÍA_OCUPACIONAL             0
CANTIDAD_DE_TRABAJOS              0
INTENSIDAD_DE_LA_OCUPACIÓN        0
ANTIGUEDAD_EN_OCUPACIÓN           0
MONTO_DE_INGRESO_TOTAL            0
DECIL_DE_INGRESOS                 0
INGRESO_TOTAL_FAMILIAR            0
DECILES_SEGÚN_INGRESO_FAMILIAR    0
DESCUENTO_POR_JUBILACIÓN          0
ESTADO_DE_OCUPACIÓN               0
dtype: int64


Como se estimo en las conclusiones de la exploración de los datos, no hay valores nulos en ninguna de las columnas. 

In [75]:
#Comprobación de la columna edad
estado.AÑOS.describe()

count    19014.000000
mean        41.140160
std         12.986582
min         13.000000
25%         30.000000
50%         41.000000
75%         51.000000
max         99.000000
Name: AÑOS, dtype: float64

In [89]:
estado.to_excel("DATA/ocupados.xlsx")

La base queda con todas sus columnas y sus valores etiquetados. Cabe aclarar que, por supuesto, no fueron los valores de todas las columnas etiquetadas, solamente aquellas que correspondían a variables categóricas. Además, a los valores de ESTADO_DE_OCUPACIÓN no los etiqueté porque fue la variable elegida para recortar la base, y no será usada en la próxima sección. 
No hizo falta eliminar el caso que tenía de edad -1 porque al hacer el recorte de la base por ocupados quedó fuera. 
Queda una base de 16 columnas, sin valores nulos y con 19014 casos (correspondientes a las personas encuestadas ocupadas). 

# 4. Presentación de datos
Vamos a presetar nuestros datos para responder las preguntas definidas en el objetivo del análisis. Poner gráficos y poner qué nos van diciendo 

# 5 Conclusiones, qué estaba buscando, qué encontres 

# 6 Referencias
links a recursos que utilizamos y que nos sirvieron para análisis